File: Codigo_Fonte_pascal/lista1/lista1_13.pas
program triangulo;

var
   a, b, c : real;

begin
     writeln('Digite os comprimentos dos lados do triangulo');
     readln(a);
     readln(b);
     readln(c);
     if ((b - c < a) and (a < b + c)) and ((a - c < b) and (b < a + c)) and ((a - b < c) and (c < a + b)) then
     begin
          //writeln('Forma triangulo');
          if ((a = b) and (b = c) ) then
             writeln('Triangulo equilatero')
          else
          begin
               if ((a <> b) and (b <> c) and (a <> c) )then
                  writeln('Triangulo escaleno')
               else
                   writeln('Triangulo isosceles');
          end;
     end
     else
          writeln('Nao forma triangulo');
     readln;
end.

File: Codigo_Fonte_c/Lista 1/lista1_13.c
#include <stdio.h>
#include <stdlib.h>

int main(){
	float a, b, c;
	scanf("%f", &a);
	scanf("%f", &b);
	scanf("%f", &c);
	if ((b - c < a && a < b + c) && (a - c < b && b < a + c) && (a - b < c && c < a + b)){
		if (a == b && b == c){
			printf("Triangulo equilatero\n");
		}else{
			if (a != b && b != c && a != c){
				printf("Triangulo escaleno\n");
			}else{
				printf("Triangulo isosceles\n");
			}
		}
		
	}else{
		printf("Nao forma triangulo\n");
	}

	return 0;
}
